catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -64,13 +66,15 @@
 	public static final String MEDIA_FILE_PATH = "media_file_path";
 	public static final String CALLING_ACTIVITY = "calling_activity";
 	private static final String FILENAME_TAG = "fname=";
+	private ProgressDialog progressCircle;
 
 	private WebView webView;
 	private boolean callMainMenu = false;
 	private String url;
 	private String callingActivity;
 	private ProgressDialog progressDialog;
 	private Intent resultIntent = new Intent();
+    private int webload_status = 0;
```

Comment:
Yea i'd initially done it with boolean, must have changed it later for some reason. I'll change it back to boolean.. Any other changes? I'll make them all together :)